package/CHANGELOG

```diff
@@ -34,6 +34,7 @@ Fixes
  * Fix deploy action to use the correct version of the pypi upload action.
 
 Enhancements
+ * Change order of arguments for adding auxiliary data (Issue #3811)
```

Comment:
This is a Change so it should go there.

Additionally, it's a breaking change (see @IAlibay 's comments) so it won't go into 2.8.0, which is unfortunate.

I'd suggest to make this clear, start a new section 
```
??/??/?? talagayev

 * 3.0.0

 Changes:
  ....

```